py/server/deephaven/jcompat.py

```diff
@@ -143,18 +145,21 @@ def j_binary_operator(func: Callable[[T, T], T], dtype: DType) -> jpy.JType:
     )
 
 
-def j_lambda(func: Callable, lambda_jtype:jpy.JType, return_dtype: DType = None):
+def j_lambda(func: Callable, lambda_jtype: jpy.JType, return_dtype: DType = None):
     """Wraps a Python Callable as a Java "lambda" type.  
     
     Java lambda types must contain a single abstract method.
     
     Args:
-        func (Callable): Any Python Callable or object with an 'apply' method that accepts the same arguments (number and type) the target Java lambda type
+        func (Callable): Any Python Callable or object with an 'apply' method that accepts the same arguments (
+            number and type) the target Java lambda type
         lambda_jtype (jpy.JType): The Java lambda interface to wrap the provided callable in
-        return_dtype (DType): The expected return type if conversion should be applied.  None (the default) does not attempt to convert the return value and returns a Java Object.
+        return_dtype (DType): The expected return type if conversion should be applied.  None (the default) does not
+            attempt to convert the return value and returns a Java Object.
     """
     coerce_to_type = return_dtype.qst_type.clazz() if return_dtype is not None else None
-    return jpy.get_type('io.deephaven.integrations.python.JavaLambdaFactory').create(lambda_jtype.jclass, func, coerce_to_type)
+    return jpy.get_type('io.deephaven.integrations.python.JavaLambdaFactory').create(lambda_jtype.jclass, func,
+                                                                                     coerce_to_type)
 
 
 def to_sequence(v: Union[T, Sequence[T]] = None) -> Sequence[Union[T, jpy.JType]]:
```

Comment:
Does a similar change need to happen in the client?  e.g. `pydeephaven_ticking/util.py`?